examples/real_function_space.py

```diff
@@ -115,26 +115,60 @@ def u_exact(x):
 
 # We can now assemble the matrix and vector
 
-A = dolfinx.fem.petsc.assemble_matrix_block(a)
+try:
+    A = dolfinx.fem.petsc.assemble_matrix_block(a)
+except AttributeError:
+    A = dolfinx.fem.petsc.assemble_matrix(a, kind="mpi")
 A.assemble()
-b = dolfinx.fem.petsc.assemble_vector_block(L, a, bcs=[])
+
+
+# On the main branch of DOLFINx, the `assemble_vector` function for blocked spaces has been rewritten to reflect how
```

Comment:
Maybe instead of writing *on the main branch* we can write *for dolfinx version greater than 0.9* or something along these lines?

Reply:
Well, but that is not strictly true, as there is something in between 0.9 and the commit that introduces this.